toolchain/sem_ir/dump.cpp

```diff
@@ -255,12 +242,11 @@ LLVM_DUMP_METHOD auto Dump(const File& file, SpecificId specific_id) -> void {
 LLVM_DUMP_METHOD auto Dump(const File& file,
                            SpecificInterfaceId specific_interface_id) -> void {
   const auto& interface = file.specific_interfaces().Get(specific_interface_id);
-  llvm::errs() << specific_interface_id << '\n';
-  llvm::errs() << "  - interface: ";
-  DumpNoNewline(file, interface.interface_id);
-  llvm::errs() << '\n';
-  llvm::errs() << "  - specific_id: ";
-  Dump(file, interface.specific_id);
+  llvm::errs() << specific_interface_id << '\n'
+               << "  - interface: "
```

Comment:
This happens to wrap the way I'd want, but it may not be stable. I think I would make a new statement per `\n` still.